src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -379,12 +380,15 @@ public void testSnapshotBuilder() throws Exception {
 
     // Test getConcepts
     var getConceptResponse = dataRepoFixtures.getConcepts(steward, datasetId, 2);
-    List<String> getConceptNames =
-        getConceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
-    assertThat(
-        "expected concepts are returned",
-        getConceptNames,
-        containsInAnyOrder("concept1", "concept3"));
+    var concepts = getConceptResponse.getResult();
+
+    var concept1 = concepts.get(0);
+    var concept3 = concepts.get(1);
+
+    assertThat(concept1.getId(), CoreMatchers.is(equalTo(1)));
```

Comment:
`is(equalTo())` is redundant. You can either say `is(1)` or `equalTo(1)`. Also I would statically import `is()` here.
```suggestion
    assertThat(concept1.getId(), is(1));
```